src/Esfa.Vacancy.Register.UnitTests/Infrastructure/GivenFrameworkCodeRepository.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Esfa.Vacancy.Register.Application.Interfaces;
+using Esfa.Vacancy.Register.Infrastructure.Repositories;
+using Esfa.Vacancy.Register.Infrastructure.Settings;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.NLog.Logger;
+
+namespace Esfa.Vacancy.Register.UnitTests.Infrastructure
+{
+    [TestFixture]
+    public class GivenFrameworkCodeRepository
```

Comment:
We could unit test the cache service

Reply:
The AzureRedisCacheService cache service is closely coupled to Redis so no way to test this as part of a unit test.  This would be a good candidate when we introduce integration tests.  The best I could do is to add tests to ensure the Framework and Standard repository call the cache.